ui/pages/swaps/prepare-swap-page/not-enough-balance.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  SEVERITIES,
```

Comment:
`SEVERITIES` has been deprecated in favor of the `Severity` enum we are actively trying to reduce it's use through [good first issue and our open source community](https://github.com/MetaMask/metamask-extension/issues/18714). Let me know if you don't see the strike through in VS Code and we can figure out what the settings are :) 

```suggestion
  Severity,
```

![Screenshot 2023-07-31 at 9 25 36 AM](https://github.com/MetaMask/metamask-extension/assets/8112138/98a94a35-c016-4979-acf3-510b1fb37e42)